rijkshuisstijl/sass/components/form/_form.scss

```diff
@@ -104,4 +105,9 @@
   &__actions .button:not(:last-child) {
     margin-right: $form-control-spacing;
   }
+
+  &__description {
+    @include text;
```

Comment:
waarom text en niet body?

Reply:
Ik zag dat we die body vooral voor div'jes e.d. gebruikte en omdat dit een paragraph is dacht ik dat text beter hier zo passen. Zal ik deze vervangen (met body) of toevoegen?

Reply:
text bevat minder styling informatie en eigenlijk puur het font. Body is bedoeld voor bodytext zoals dit (en gebruikt intern text)